src/main/java/org/tikv/common/PDClient.java

```diff
@@ -500,17 +500,15 @@ protected PDBlockingStub getBlockingStub() {
     }
     return leaderWrapper
         .getBlockingStub()
-        .withDeadlineAfter(getConf().getTimeout(), getConf().getTimeoutUnit());
+        .withDeadlineAfter(getTimeout(), getConf().getTimeoutUnit());
```

Comment:
`getTimeout(), getConf().getTimeoutUnit()` is a little strange.

both `getConf().getTimeout(), getConf().getTimeoutUnit()` and `getTimeout(), getTimeoutUnit()` are ok.